exercises/allergies/test/Tests.hs

```diff
@@ -98,3 +129,81 @@ specs = do
                                           , Shellfish
                                           , Strawberries
                                           , Tomatoes     ]
+
+            -- Property: For an arbitrary `allergen` and its `score`,
+            -- `allergies score` is a list of exactly the element
+            -- `allergen`.
+
+            it "accepts single allergens" $ forAll allergenWithScore $
+              \(allergen, score) -> allergies score == [allergen]
+
+            -- Property: For an arbitrary `score` and arbitrary `allergen`
+            -- that does not match it, that `allergen` is not an element of
+            -- `allergies score`.
+
+            it "rejects mismatching allergens" $ forAll complementWithScore $
+              \(allergen, score) -> allergen `notElem` allergies score
```

Comment:
I think this particular test (`notElem`) is not necessary, because testing for exact list equality in `allergies` already precludes this possibility. So I would think either "accepts single allergens" needs to change to using `elem` (and this test remains), or this test should be removed

Reply:
I was quite tired when I read this, so I've waited until now to address this.

I understand the difference and agree that this test is redundant.